src/main/java/kr/pickple/back/crew/service/CrewService.java

```diff
@@ -68,6 +73,24 @@ public CrewProfileResponse findCrewById(final Long crewId) {
         return CrewProfileResponse.fromEntity(crew, confirmedMemberDtos);
     }
 
+    public List<CrewProfileResponse> findCrewByAddress(final String addressDepth1, final String addressDepth2, final int page, final int size) {
+        final AddressDepth1 depth1 = addressService.findByExistAddressDepth1(addressDepth1);
+        final AddressDepth2 depth2 = addressService.findByExistAddressDepth2(addressDepth2, depth1);
```

Comment:
`p1`

위에서 말씀드린 것 처럼 재사용성을 위해 아래와 같이 코드 변경해서 작성해주시면 좋을 것 같습니다! 

```suggestion
        final MainAddressResponse mainAddressResponse = addressService.findMainAddressByNames(addressDepth1, addressDepth2);
```